src/core2/1E6F080.c

```diff
@@ -1,6 +1,9 @@
 #include "core2/1E6F080.h"
 
-#pragma GLOBAL_ASM("asm/nonmatchings/core2/1E6F080/func_80095790.s")
+s32 func_80095790() 
+{
+return 0xAC;
```

Comment:
Missing indentation one some of these matches, should be easy to fix with a find replace for `\nreturn` to `\n    return`.